TramsDataApi/Factories/Concerns/Decisions/DecisionFactory.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Linq;
+using TramsDataApi.DatabaseModels.Concerns.Case.Management.Actions.Decisions;
+using TramsDataApi.RequestModels.Concerns.Decisions;
+
+namespace TramsDataApi.Factories.Concerns.Decisions
+{
+    [Obsolete("This is planned to be moved into the Concerns Casework API. If it is accessed by other APIs, please let the Concerns team know.")]
+    public class DecisionFactory : IDecisionFactory 
+    {
+        public Decision CreateDecision(int concernsCaseId, CreateDecisionRequest request)
+        {
+            var decisionTypes = request.DecisionTypes.Select(x => new DecisionType(x)).ToArray();
+
+            return new Decision(concernsCaseId, request.CrmCaseNumber, request.RetrospectiveApproval,
```

Comment:
In contrast to the belt-and-braces validation in the decision response factory, there is no validation here I think. Again, would just be good to agree an approach

Reply:
It's slightly different in that it's not as simple as being a positive number. So the DecisionType encapsulates the real logic. And the Decision will eventually do some validation too. 
If anything I'd remove it from the previous factory and just let it pass through to decision response.
The factory for decision response is really a waste of time as it hides 1 real line of code. Unfortunately was necessary in order to test properly because of other limitations caused by how the api code works in general